cypress/e2e/types/constants.ts

```diff
@@ -118,6 +118,7 @@ export const general = "General";
 export const instanceName = "Instance name";
 export const cantDeleteJiraAlert =
     "Danger alert:This instance contains issues associated with applications and cannot be deleted";
+export const singleApplication = "Single application";
```

Comment:
This can go in Issues view . WDYT?

Reply:
No, view is for selectors. This is just a string, so it goes to constants.